### PR TITLE
media-plugins/gst-plugins-webp: New package to add WebP support to gstreamer

### PR DESCRIPTION
**Pull request by kripton**
Bug: https://bugs.gentoo.org/836666
Package-Manager: Portage-3.0.30, Repoman-3.0.3
Signed-off-by: Jannis Achstetter <kripton@kripserver.net>

### CODE REVIEW

**Comment by mattst88** [comment]
cc: @i-garrison

**Comment by leio** [comment]
This needs to be a split package, not a USE flag on gst-plugins-bad

**Comment by kripton** [comment]
@leio : Ok, thanks for the info. You mean something like https://gitweb.gentoo.org/repo/gentoo.git/tree/media-plugins/gst-plugins-zbar/gst-plugins-zbar-1.20.1.ebuild ? I'll give it a try then

**Comment by kripton** [comment]
I updated this PR to add a new gstreamer plugin instead of enhancing gst-plugins-bad. v1.20.1 has been tested, v1.18.x is just added the same way without being tested by me

File: media-plugins/gst-plugins-webp/gst-plugins-webp-1.18.6.ebuild [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+GST_ORG_MODULE=gst-plugins-bad
+
+inherit gstreamer-meson
+
+DESCRIPTION="WebP image format support for GStreamer"
+KEYWORDS="~amd64"
+IUSE=""
```
Comment (i-garrison) [inline media-plugins/gst-plugins-webp/gst-plugins-webp-1.18.6.ebuild:11]: You probably still do not need empty IUSE here.

**Comment by i-garrison** [comment]
I believe this PR should work as advertised and matches general pattern of gstreamer plugins found via "Feature options for plugins that need external deps" section of meson options.

**Pull request closed without merging**